docs/sources/panels-visualizations/configure-standard-options/index.md

```diff
@@ -121,6 +121,7 @@ Select one of the following palettes:
 | **Single color**                | Specify a single color, useful in an override rule                                                                                                       |
 | **From thresholds**             | Informs Grafana to take the color from the matching threshold                                                                                            |
 | **Classic palette**             | Grafana will assign color by looking up a color in a palette by series index. Useful for Graphs and pie charts and other categorical data visualizations |
+| **Classic palette by name**     | Grafana will assign color based on the name of the series. Useful when the series names to be visualized depend on the available data.                   |
```

Comment:
```suggestion
| **Classic palette (by series name)**     | Grafana will assign color based on the name of the series. Useful when the series names to be visualized depend on the available data.                   |
```

Reply:
this one will require a pass through by prettier

Reply:
Done